script/Ciel/Day 4/04_01_CIEL04_6_2.txt

```diff
@@ -147,7 +147,7 @@ I may have surreptitiously run away... but I came back, didn't I?!
 -- Page 152, Offset 8558.
 -- いまさら弁明しても後の祭りだが、しないで殺されるのは逃げ出すよりみっともないので全力で言い訳する。
 It's already too late to try and explain myself,
- but if I end up getting killed without even %{i}attempting%{/i} to that would be even more shameful than having run away in the first place,
+ but if I end up getting killed without even %{i}attempting%{/i} to, that would be even more shameful than having to run away in the first place,
```

Comment:
Don't need that `to`, he didn't "have to" run away, he chose to, he is simply in the state of _having run away_ (run away in the past).